<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EXFO_T100_PM_Power_Step_Test_Window</class>
 <widget class="QMainWindow" name="EXFO_T100_PM_Power_Step_Test_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="testSettingsGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Test Settings</string>
    </property>
    <widget class="QLabel" name="startPowerLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Power:</string>
     </property>
    </widget>
    <widget class="QLabel" name="endPowerLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>End Power:</string>
     </property>
    </widget>
    <widget class="QLabel" name="powerStepSizeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Step Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="dwellLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Dwell:</string>
     </property>
    </widget>
    <widget class="QLabel" name="wavelengthLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Wavelength:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="startPowerEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="endPowerEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="powerStepSizeEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dwellEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="wavelengthEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="startPowerUnitLabel">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>55</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(dBm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="endPowerUnitLabel">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>110</y>
       <width>55</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(dBm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="powerStepSizeUnitLabel">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>55</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(dBm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="dwellUnitLabel">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>250</y>
       <width>55</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(s)</string>
     </property>
    </widget>
    <widget class="QLabel" name="wavelengthUnitLabel">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>320</y>
       <width>55</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>(nm)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="testTitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EXFO T100 and Power Meter - Power Meter Step Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTestButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>670</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showTestDescriptionButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Test Description</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="availableT100sGroupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>181</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="powerMeterSlotsGroupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>231</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T100 Module to Power Meter Slot Pairing:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
